Source/WebCore/dom/ViewTransitionTypeSet.cpp

```diff
@@ -52,9 +52,11 @@ void ViewTransitionTypeSet::initializeSetLike(DOMSetAdapter& setAdapter) const
 void ViewTransitionTypeSet::clearFromSetLike()
 {
     std::optional<Style::PseudoClassChangeInvalidation> styleInvalidation;
-    if (m_document.documentElement()) {
+    if (!m_document)
```

Comment:
I was gonna say we should `ASSERT(m_document)`, since it's a precondition that m_document is always valid... but I guess it's not always the case, judging from the failing tests. Looking at the falling tests, seems like the Document went away before JSC garbage collected ViewTransition (which contains a ViewTransitionTypeSet) away.

Reply:
Yes, that's what I gathered from my debugging.